sources/osg/Vec3.js

```diff
@@ -125,8 +125,12 @@ var Vec3 = {
         r[ 1 ] = a[ 1 ] + ( b[ 1 ] - a[ 1 ] ) * t;
         r[ 2 ] = a[ 2 ] + ( b[ 2 ] - a[ 2 ] ) * t;
         return r;
+    },
+    equal: function ( a, b ) {
+        if ( a === b ) return true;
+        if ( a === undefined || b === undefined ) return false;
```

Comment:
imho better to *not* do any check, why would it be undefined or even null???
The check should be done before the call and handled elsewhere.